assets/less/zenodo-rdm/collections/menu.overrides

```diff
@@ -0,0 +1,10 @@
+/*
+ *   Copyright (C) 2021-2022 CERN.
+ *
+ * Invenio App RDM is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+#invenio-nav.ui.menu .ui.menu .item a:not(.ui.button) {
```

Comment:
double .ui.menu class, in addition we should less notation

Reply:
It's double because there are nested `.ui.menu`'s. If I leave it out on `#invenio-nav` it's not specific enough to be overridden.